spec/spec_helper.rb

```diff
@@ -5,7 +5,7 @@
 require "simplecov"
 SimpleCov.start
 
-require "active_record"
+require "sidekiq_publisher"
```

Comment:
I had an issue where I was requiring `activerecord` and it was only caught locally in specs, so changed this to require the root file so that it is tested explicitly.